zppy/utils.py

```diff
@@ -140,9 +143,135 @@ def getComponent(input_files):
 
 
 # -----------------------------------------------------------------------------
-def submitScript(scriptFile, dependFiles=[], export="ALL"):
+# Bundles
+class Bundle(object):
```

Comment:
I ended up going with a class over a dictionary after all. There was a lot of information to process.